implementations/tests/tests_df/test_open_issue_age_github.py

```diff
@@ -41,7 +41,7 @@ def test_compute(self):
         """
 
         open_issue_age = OpenIssueAgeGitHub(self.items)
-        expected_mean = 1132.0
+        expected_mean = 1179.0
         mean_age = open_issue_age.compute()
```

Comment:
This constant change is required fpr the tests of Open Issue Age since age is being computed like so:
```
flat['open_issue_age'] = (datetime.now() - flat['created_date']).days
```
In a following update, I'll make `expected_mean` dynamic, making use of `datetime.now()`.

Reply:
Fixed in #247